### DISCUSSION TITLE
Additional entities for HA dashboard

### DISCUSSION THREAD

**Discussion by charlesvandortmond** [lavinir/hassio-onedrive-backup#213] [Ideas]
Hello,

I can only select 4 parameters for the Home Assistant dashboard, I would like to also show the date/time of the last update. Is this possible?

Regards
Charles

![onedrive](https://github.com/lavinir/hassio-onedrive-backup/assets/74618517/d4094c1d-07a8-40ba-86d9-16346aaf21c2)

**Comment by lavinir** [comment]
The additional data points are exposed as attributes. Look at the attributes for example of the OneDrive Backup entity.
